package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "PORT=3001 react-scripts start ",
-    "build": "react-scripts build",
+    "build": "react-scripts build && echo '/* /index.html 200' | cat >build/_redirects",
```

Comment:
what does this do?

Reply:
This helps netlify redirect to correct page. Instead of rendering an error.